docs/main/docs/components/become-an-executor.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 2
+---
+
+# How to Become an Executor
+
+This documentation covers how to become an Executor on the t3rn network. It assumes that you have accounts with the correct funds for the desired blockchains, and that you have Substrate installed. We're using subkey to generate keys but feel free to skip this part if you already have your keys or using another tool to generate some.
+
+## Step 1 - Clone repo
+
+Clone the [t3rn repo](https://github.com/t3rn/t3rn/tree/development/client/packages/executor): `git clone git@github.com:t3rn/t3rn.git`.
+
+<p align="center">
+    <img height="150" src="/img/clone-executor-repo.png?raw=true"/>
+</p>
+
+## Step 2 - Install dependencies
+
+`pnpm install` in the above directory (t3rn/client/packages/executor/).
+
+<p align="center">
+    <img height="150" src="/img/install-executor-dep.png?raw=true"/>
+</p>
+
+## Step 3 - Configure your Executor
```

Comment:
I already commented on previous PR that direnv is optional and used by us.
https://github.com/t3rn/t3rn/pull/1403

Reply:
Removed direnv from docs